### PR TITLE
Exclude devDependencies in gulpfile. Deprecate bower. Issue #845

### PR DESCRIPTION
**Pull request by SergeyBondarenko**
part of https://github.com/sirensolutions/kibi-enterprise/issues/845

### CODE REVIEW

**Comment by SergeyBondarenko** [comment]
jenkins test it

File: package.json [hunk 1]
```diff
@@ -13,13 +13,14 @@
...
     "start": "gulp dev",
-    "postinstall": "bower install",
     "precommit": "gulp lint",
     "build": "gulp build"
   },
   "dependencies": {
     "kibiutils": "sirensolutions/kibiutils#1.4.0",
-    "kibi-icons": "sirensolutions/kibi-icons#1.0.0"
+    "kibi-icons": "sirensolutions/kibi-icons#1.0.0",
+    "lodash": "3.10.1",
```
Comment (szydan) [inline package.json]: I think lodash is not needed as it is already present in kibana/kibi
Could you try remove it and test if everything still works

Comment (SergeyBondarenko) [inline package.json]: Deleted

**Comment by szydan** [comment]
@SergeyBondarenko  Please resolve the conflict

**Pull request merged**